ignition/src/IgnitionRobot.cpp

```diff
@@ -1006,6 +1007,35 @@ bool IgnitionRobot::resetJoint(const gympp::Robot::JointName& jointName,
     return true;
 }
 
+bool IgnitionRobot::addExternalWrench(const gympp::Robot::LinkName& linkName,
+                                      const std::array<double, 3>& force,
+                                      const std::array<double, 3>& torque)
+{
+    LinkEntity linkEntity = pImpl->getLinkEntity(linkName);
+    assert(linkEntity != ignition::gazebo::kNullEntity);
```

Comment:
The user of assert here means that if the linkname is wrong, the Python interpreter will crash if the plugin has been compiled in debug, or it will silently fail if it is compiled in Release. Could it make sense to propagate the error instead?
```suggestion
    if (linkEntity != ignition::gazebo::kNullEntity) {
        return false;
    }
```

Reply:
This makes sense, and it is a pattern that was used in the same file:

https://github.com/robotology/gym-ignition/blob/7292f5069aa29750ddcb00fd0973f4d070345115/ignition/src/IgnitionRobot.cpp#L980-L983

However, for some reason there are other methods that use the assert. I will change this line here, and will address the other ones in a separated PR. Thanks!

Reply:
Check this diff for the changes https://github.com/robotology/gym-ignition/compare/a98480d2d0ce9647f81ea7de49263d8fe2281611..af4c9d750f953f87da4eeede155e9ab9aa1ea43d

Reply:
> Check this diff for the changes https://github.com/robotology/gym-ignition/compare/a98480d2d0ce9647f81ea7de49263d8fe2281611..af4c9d750f953f87da4eeede155e9ab9aa1ea43d

Perfect!